plinth/views.py

```diff
@@ -208,8 +208,19 @@ def get_context_data(self, **kwargs):
         """Return the context data rendering the template."""
         context = super(SetupView, self).get_context_data(**kwargs)
         context['setup_helper'] = self.kwargs['setup_helper']
+        context['dpkg_is_active'] = self.is_dpkg_active()
         return context
 
+    def is_dpkg_active(self):
+        output = plinth.actions.superuser_run('is_dpkg_active')
+        output = output.rstrip()
+        if output == 'True':
+            return True
+        elif output == 'False':
+            return False
+        else:
+            raise ActionError("Unexpected output from is_dpkg_active")
+
```

Comment:
Thank you for the tip; I've installed it into Emacs now, but I'm knew to even the editor. Some things are shown, however that particular style isn't. Will check and fix that soon, now I start appreciating code styles.